dists/go-ipfs/versions

```diff
@@ -60,6 +60,8 @@ v0.10.0
 v0.11.0-rc1
 v0.11.0-rc2
 v0.11.0
+v0.11.1
```

Comment:
I'm not sure if it matters where in this file `v0.11.1` goes, i.e. if it should be in semver order or chronological order.

The tooling uses chronological ordering but I can't tell if that's a bug or not.

Reply:
If someone cares, there is `date` field in `dist.json` created during the build, it indicates when release was published.